operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -416,4 +420,293 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", ResourceType.Value.LOCATION, searchParameters, allowedIncludes); 
+        } else if (compartmentMemberType.equals(ResourceType.Value.SPECIMEN.value())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "collector", ResourceType.Value.PRACTITIONER, searchParameters, allowedIncludes);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", ResourceType.Value.LOCATION, searchParameters, allowedIncludes);
+        } else if (compartmentMemberType.equals(ResourceType.Value.SUPPLY_DELIVERY.value())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "receiver", ResourceType.Value.PRACTITIONER, searchParameters, allowedIncludes);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "supplier", ResourceType.Value.ORGANIZATION, searchParameters, allowedIncludes);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "supplier", ResourceType.Value.PRACTITIONER, searchParameters, allowedIncludes);
+        } else if (compartmentMemberType.equals(ResourceType.Value.SUPPLY_REQUEST.value())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "requester", ResourceType.Value.ORGANIZATION, searchParameters, allowedIncludes);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "requester", ResourceType.Value.PRACTITIONER, searchParameters, allowedIncludes);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", ResourceType.Value.LOCATION, searchParameters, allowedIncludes);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", ResourceType.Value.ORGANIZATION, searchParameters, allowedIncludes);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "supplier", ResourceType.Value.ORGANIZATION, searchParameters, allowedIncludes);
+        } else if (compartmentMemberType.equals(ResourceType.Value.VISION_PRESCRIPTION.value())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "prescriber", ResourceType.Value.PRACTITIONER, searchParameters, allowedIncludes);
+        } 
+            
+        // BodyStructure, ResearchSubject, and RelatedPerson have no references to:
+        // Location, Medication, Organization, and Practitioner
+        
+        // DeviceUseStatement, EnrollmentRequest, FamilyMemberHistory, ChargeItem, 
+        // ImmunizationEvaluation, ImmunizationRecommendation, MolecularSequence, 
+        // and NutritionOrder have no search parameters that reference: 
+        // Location, Medication, Organization, and Practitioner
+    }
+    
+    private void addSearchParameterIfNotExcluded(String compartmentMemberType, String code, 
+        ResourceType.Value subResourceType, MultivaluedMap<String, String> searchParameters, 
+        List<String> allowedIncludes) {
```

Comment:
minor style thing...we double-indent method declaration line spillover to visually distinguish from the first statement in the method:
```suggestion
            ResourceType.Value subResourceType, MultivaluedMap<String, String> searchParameters, 
            List<String> allowedIncludes) {
```

Reply:
Committed change.